src/main.cc

```diff
@@ -1,14 +1,38 @@
-#include <iostream>
-#include <ctime>
+#include "settings_menu.h"
+#include "settings_item.h"
+#include "menu_item.h"
+#include "main_menu.h"
 
 int main() 
 {
-  int count = 0;
-  int start = std::time(nullptr);
+  roadrun::MenuItem meme = roadrun::MenuItem::kStartGame;
+  
+  roadrun::MainMenu *p = new roadrun::MainMenu();
+  roadrun::SettingsMenu *d = new roadrun::SettingsMenu();
 
-  while(std::time(nullptr) != start + 1)
+  // this can change
+  while(1)
   {
-    std::cout << count++ << std::endl;
+    p->PrintMenu();
+    roadrun::MenuItem meme = p->GetUserChoice();
+
+    std::cout << meme << std::endl;
+    sleep(1);
```

Comment:
So I commented this out and it still pauses for roughly a second after selecting a menu option. Is this expected behavior?

Reply:
Nevermind, found the other `sleep()` calls